scripts/installGUI

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+export WITH_MESH_POINT=false
+export WITH_WIFI_AP=false
+export WITH_IPFS=false
+export WITH_PROMETHEUS_NODE_EXPORTER=false
+export WITH_PROMETHEUS_SERVER=false
+export WITH_GRAFANA=false
+export WITH_EXTRA_TOOLS=false
+export WITH_H_DNS=false
+export WITH_H_NTP=false
+export WITH_FAKE_HWCLOCK=false
+
+export SET_MESH_POINT=off
+export SET_WIFI_AP=off
+export SET_IPFS=off
+export SET_PROMETHEUS_NODE_EXPORTER=off
+export SET_PROMETHEUS_SERVER=off
+export SET_GRAFANA=off
+export SET_EXTRA_TOOLS=off
+export SET_H_DNS=off
+export SET_H_NTP=off
+export SET_FAKE_HWCLOCK=off
+
+sudo apt-get install dialog
+
+dialog --backtitle "TOMesh Prototype Node" --title "Select feature starting point (Customize next)" --menu "Presets" 20 70 10 \
+        BASIC "Basic" \
+        HYPE  "Basic With Hyperboria" \
+        FULL  "Install Everything" \
+        CANCEL "Cancel" \
+2> /tmp/answer
+
+res=$(cat /tmp/answer)
+rm -rf /tmp/answer
+
+if [ "$res" == "CANCEL" ]; then
+    exit 0
+fi
+if [ "$res" == "BASIC" ]; then
+    export SET_MESH_POINT=on
+    export SET_WIFI_AP=on
+fi
+
+if [ "$res" == "HYPE" ]; then
+    export SET_MESH_POINT=on
+    export SET_WIFI_AP=on
+    export SET_PROMETHEUS_NODE_EXPORTER=on
+    export SET_EXTRA_TOOLS=on
+    export SET_H_DNS=on
+    export SET_H_NTP=on
+    export SET_FAKE_HWCLOCK=on
+
+fi
+
+
+if [ "$res" == "FULL" ]; then
+    export SET_MESH_POINT=on
+    export SET_WIFI_AP=on
+    export SET_IPFS=on
+    export SET_PROMETHEUS_NODE_EXPORTER=on
+    export SET_PROMETHEUS_SERVER=on
+    export SET_GRAFANA=on
+    export SET_EXTRA_TOOLS=on
+    export SET_H_DNS=on
+    export SET_H_NTP=on
+    export SET_FAKE_HWCLOCK=on
+fi
+
+dialog --backtitle "TOMesh Prototype Node" --title "Select Features" --checklist "Features:" 20 70 10 \
+        "WITH_MESH_POINT" "Mesh Point" $SET_MESH_POINT \
```

Comment:
Thinking the first column should be the title case label instead of the `export` name. e.g.
`"Mesh Point" "Allow peering through an 802.11s interface" $SET_MESH_POINT \`

Reply:
Action is not by choice

The first column is what the dialog app sets the value the second column is the description.

Reply:
The other issue is that some of the export are really long, like the prom node exporter, if we can make it smaller we can allow for more room for the friendly names.